pennylane/ops/op_math/evolution.py

```diff
@@ -147,7 +147,7 @@ def generator(self):
                 f"The operator coefficient {self.coeff} is not imaginary; the expected format is exp(-ixG)."
                 f"The generator is not defined."
             )
-        return self.base
+        return -1 * self.base
 
```

Comment:
How is it possible that this was not causing issues before?

Reply:
🤷🏻‍♂️